### ISSUE TITLE
How to change font size?

### ISSUE THREAD

**Issue by ChenXiaoman** [nus-cs2103-AY1718S1/forum#179]
![image](https://user-images.githubusercontent.com/25006870/32335572-263226fe-bfbb-11e7-987e-79dc8f13fdde.png)
Check out the userGuide [here](https://github.com/ChenXiaoman/addressbook-level4/blob/master/docs/UserGuide.adoc#change-font-size-code-fs-code-since-v1-3)
Check out the implementation [here](https://github.com/ChenXiaoman/addressbook-level4/blob/master/docs/DeveloperGuide.adoc#customize-font-size-mechanism)
Enjoy :)

**Comment by damithc** [comment]
@ChenXiaoman for this to be counted as a CLI command, provide a command for the same functionality. e.g. The command `+` increases font size.

**Comment by ChenXiaoman** [comment]
Yes, user can change the font size by giving command specifying the name of the exact font size which they want the app to be changed to. E.g.` fs xl`. Or they can use "+" or "-" symbol to increase or decrease the current font size. E.g. `fs +`

Pressing the "+" button equals to giving `fs +` command and similarly for "-" button.
